docs/content/docs/updates/v1.0.0.md

```diff
@@ -24,7 +24,7 @@ We are now surfacing errors on FluidContainer "disposed" event, so the applicati
 
 ## fluid-framework
 
--   Add the missing [ConnectionState]({{< packageref "container-loader" "v1" >}}#connectionstate-enum) export to `fluid-framework`
+-   Add the missing [ConnectionState]({{< packageref "container-loader" "v1" >}}connectionstate-enum) export to `fluid-framework`
```

Comment:
I don't think this change is right - `connectionstate-enum` is a heading on the page, right?

Reply:
Oh, we made it so enums get their own pages, didn't we? In that case, we should be able to update this to use `apiref` instead of `packageref`, and point to the enum more directly.

Reply:
I can only say: the separate page exists, the header does not. Not familiar with what the `apiRef`/`packageRef` part means 😄. Should I keep the change, or apply that one somewhere else?

Reply:
I think you can replace the link with...
```suggestion
-   Add the missing [ConnectionState]({{< apiref "container-loader" "ConnectionState" "enum" "v1" >}}) export to `fluid-framework`
```

Though you will want to verify

Reply:
Validated the apiref version locally.